src/Admin/JecouteSurveyAdmin.php

```diff
@@ -49,13 +50,21 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
     {
         $datagridMapper
             ->add('name')
-            ->add('author', null, [
+            ->add('author', ModelAutocompleteFilter::class, [
                 'label' => 'Auteurs',
+                'field_options' => [
+                    'property' => 'lastName',
+                ],
                 'show_filter' => true,
             ])
-            ->add('author.referentTags', null, [
+            ->add('author.referentTags', ModelAutocompleteFilter::class, [
                 'label' => 'Zones',
                 'show_filter' => true,
+                'field_options' => [
+                    'model_manager' => $this->getModelManager(),
+                    'admin_code' => $this->getCode(),
+                    'property' => 'name',
```

Comment:
What about adding `'minimum_input_length' => 2,` to be able to search by department number ? 
**before:**
![screenshot from 2019-01-31 12-48-20](https://user-images.githubusercontent.com/8271347/52052439-ad837700-2556-11e9-8ec5-c93b8973e81b.png)

**after:**
![screenshot from 2019-01-31 12-49-12](https://user-images.githubusercontent.com/8271347/52052447-b411ee80-2556-11e9-901c-b353b0cb8ce4.png)

Reply:
fixed